app/src/main/kotlin/com/wire/android/ui/common/ClassifiedIndicatorBanner.kt

```diff
@@ -0,0 +1,85 @@
+package com.wire.android.ui.common
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.wire.android.R
+import com.wire.android.ui.theme.wireTypography
+import com.wire.kalium.logic.feature.conversation.ClassifiedType
+
+@Composable
+fun ClassifiedIndicatorBanner(
```

Comment:
It reads a bit odd to me. Maybe `ClassificationLevelBanner` or `SecurityClassificationBanner` would be more self explanatory?

Reply:
Applied ! :D